library/analytics/src/main/java/org/cru/godtools/analytics/firebase/FirebaseAnalyticsService.kt

```diff
@@ -80,7 +80,11 @@ class FirebaseAnalyticsService @MainThread @Inject internal constructor(
 
     @MainThread
     private fun handleScreenEvent(event: AnalyticsScreenEvent) {
-        currentActivity?.let { firebase.setCurrentScreen(it, event.screen, null) }
+        val bundle = Bundle()
+        bundle.putString(FirebaseAnalytics.Param.SCREEN_NAME, event.screen)
+        currentActivity?.let {
+            firebase.logEvent(FirebaseAnalytics.Event.SCREEN_VIEW, bundle)
+        }
```

Comment:
I don't know if this is the best way to do this, but this is how they did it in the documentation to migrating from ```setScreenName```. [Here's the doc](https://firebase.googleblog.com/2020/08/google-analytics-manual-screen-view.html).

Reply:
This can be simplified a bit, you don't actually use`currentActivity` anymore.

Reply:
And following that up, is`currentActivity` used elsewhere?

Reply:
Here are the only other spots where it is used: https://github.com/CruGlobal/godtools-android/blob/45d4bdb9967c77007dfb781fee7e4599edd901c0/library/analytics/src/main/java/org/cru/godtools/analytics/firebase/FirebaseAnalyticsService.kt#L58-L66

Reply:
so those callbacks set/clear `currentActivity`, which is no longer necessary since you aren't using `currentActivity`. those calls can be removed, and then the `ActivityLifecycleCallbacks` interface can be removed from `FirebaseAnalyticsService` because it no longer does anything.